Dockerfile

```diff
@@ -0,0 +1,17 @@
+# This Dockerfile places the python server in /opt/spinnaker-monitoring/bin, and expects
+# config to be mounted in /opt/spinnaker-monitoring/config, which includes the 
+# spinnaker-monitoring.yml file.
+FROM python:2.7
+
+COPY spinnaker-monitoring /opt/spinnaker-monitoring/bin
+
+COPY requirements.txt /opt/spinnaker-monitoring/requirements.txt
+
+WORKDIR /opt/spinnaker-monitoring
+
+RUN pip install -r requirements.txt
+
+ENV PYTHONWARNINGS "once"
+
+CMD ["python", "/opt/spinnaker-monitoring/bin"]
+ENTRYPOINT ["--config_dir", "/opt/spinnaker-monitoring/config", "monitor"]
```

Comment:
In the PR I sent you  --config_dir no longer exists and the flag is not needed. However if you are going to mount the spinnaker-monitoring.yml file into a directory other than /opt/spinnaker-monitoring then you need a --config parameter that is the path to the .yml file (not to the directory containing it).

The directory containing the source .yml files should be mounted as /opt/spinnaker-monitoring/registry, or pass a --registry parameter specifying the path you mounted it to. The directory contains the yml files directly, there is no longer a "sources" subdirectory.

Fix the instructions at the top of the file.